### PR TITLE
Training PR

### PR DESCRIPTION
**Pull request by matthewblazewicz-kcom**
Dear Gergely,

I hope this find you well...

### CODE REVIEW

File: training [hunk 1]
```diff
@@ -0,0 +1 @@
+A new training file.
```
Comment (GergelyStyevko) [inline training:1]: This is Great!

**Comment by mblazewicz** [comment]
No.

**Comment by GergelyStyevko** [comment]
I am missing a file. Can you add it, please?

**Comment by matthewblazewicz-kcom** [comment]
No way!

**Comment by GergelyStyevko** [comment]
Well let us see what we can do.

**Comment by GergelyStyevko** [comment]
Done.

**Pull request merged**